exporters/otlp/otlptrace/internal/otlpconfig/options_test.go

```diff
@@ -92,10 +92,55 @@ func TestConfigs(t *testing.T) {
 		{
 			name: "Test With Endpoint",
 			opts: []otlpconfig.GenericOption{
-				otlpconfig.WithEndpoint("someendpoint"),
+				otlpconfig.WithEndpoint("https://127.0.0.1:1234"),
 			},
 			asserts: func(t *testing.T, c *otlpconfig.Config, grpcOption bool) {
-				assert.Equal(t, "someendpoint", c.Traces.Endpoint)
+				assert.Equal(t, "127.0.0.1:1234", c.Traces.Endpoint)
+			},
+		},
+		{
+			name: "Test With Endpoint without scheme",
+			opts: []otlpconfig.GenericOption{
+				otlpconfig.WithEndpoint("127.0.0.1:4318"),
```

Comment:
I think we can assign this localhost address `127.0.0.1:4318` in a variable and then can use it accordingly.

Reply:
I could find that the same IP address is also being used in the same manner in other test files as well. So should we do this just for this file? @ApsTomar Please share your thoughts. Thanks.

Reply:
If different IP addresses are being used at different places, then we can't amend other unit tests. We can only focus on our unit test. So, simply use a local variable to store this IP and use it here in your unit test. Also, take into consideration the scope of that variable so that you don't have to declare it again and again in every test case.

Reply:
So I checked the unit tests and it seems that there are different ports and schemes being used with this IP to test different scenarios. So if I store only `127.0.0.1` in a variable, it'll increase the unnecessary sting formatting. Also, if we declare this variable at package level and only use in our test case, it might not be consistent as all other test cases do not use it. @shubham-bansal96 , @ApsTomar if you guys still think we should make this change, please let me know. Thanks.